### PR TITLE
Update contributor docs

### PR DESCRIPTION
**Pull request by domyen**
## What I did
Updated contributor docs for brevity and clarity.

## How to test

- Is this testable with Jest or Chromatic screenshots? No
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/contribute/code.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ yarn
 yarn bootstrap --core
 ```
 
-## Sanity check
+## Run Storybook utilities
```
Comment (shilman) [inline docs/contribute/code.md:18]:
```suggestion
## Run tests & examples
```

File: docs/contribute/code.md [hunk 2]
```diff
@@ -98,13 +102,20 @@ yarn test
 
 Doing this prevents last-minute bugs and is also a great way to get your contribution merged faster once you submit your pull request. Failing to do so will lead to one of the maintainers mark the pull request with the **Work in Progress** label until all tests pass.
 
+### Target `next` branch
+
 Once the test suite finishes, it's time to commit, push and open a pull request against Storybook's `next` (default) branch. This branch is where all active development happens and is associated with the latest prerelease version (e.g., `6.3.0-alpha.25`).
 
-If your contribution focus on a bugfix and you want it featured in the next stable release, mention it in the pull request description. We'll try to get it in if it appears to be non-disruptive and fixes a critical bug.
+If your contribution focuses on a bugfix and you want it featured in the next stable release, mention it in the pull request description. We'll try to get it in if it appears to be non-disruptive and fixes a critical bug.
```
Comment (shilman) [inline docs/contribute/code.md:109]:
```suggestion
If your contribution focuses on a bugfix and you want it featured in the next stable release, mention it in the pull request description. We'll try to patch it in if it appears to be non-disruptive and fixes a critical bug.
```

**Review by shilman — APPROVED** [review]
Great changes! A couple tweaks below

**Comment by domyen** [comment]
@jonniebigodes ready for review when you get a chance

File: docs/contribute/new-snippets.md [hunk 3]
```diff
@@ -2,18 +2,20 @@
 title: 'Code snippets contributions'
 ---
 
-Contributions to the snippets used in our documentation are welcome and greatly encouraged. This page outlines how to contribute to [Storybook's](https://github.com/storybookjs/storybook) code snippets. For instructions on working with the codebase, check the [code contributions](./code.md) page, and for documentation, check the [documentation contributions](./documentation-updates.md) page.
+Add or update the code snippets in the documentation. This page outlines how the code snippets are structured.
 
 ## Documented frameworks
 
-The Storybook documentation is highly versatile and constantly updated to solve everyday scenarios. Currently, it features examples for the following UI frameworks:
+Storybook maintains code snippets for a [variety of frameworks](./../api/frameworks-feature-support). We try to keep them up to date as framework APIs evolve. But it's tricky to keep track of every API change in every framework.
```
Comment (jonniebigodes) [inline docs/contribute/new-snippets.md:9]: @domyen when you're able, add the `.md` extension to this link, to allow the proper navigation in both GitHub and the website.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@domyen one small item that needs to be updated and this should be good to go.

File: docs/contribute/new-snippets.md (hunk 3 above)
Comment (jonniebigodes) [reply docs/contribute/new-snippets.md]: Fixed with the latest commit. Marking this as solved

**Comment by jonniebigodes** [comment]
@domyen looks good to me. Self merging. @winkerVSbecks thanks for catching the broken link.

**Pull request merged**